### ISSUE TITLE
Rascal MF checker should make sure there is always a space after a `:`

### ISSUE THREAD

**Issue by DavyLandman** [usethesource/rascal-language-servers#300]
```
Manifest-Version: 0.0.1
Project-Name: test-lib
Source: src/main/rascal
Require-Libraries:

```

fails because `Require-Libraries:` is missing a space after the `:`

We should add another check to the RASCAL.MF validator

**Comment by jurgenvinju** [comment]
We might consider moving to yaml? This is so finicky..

**Comment by jurgenvinju** [comment]
But let's first do that indeed

**Comment by DavyLandman** [comment]
> We might consider moving to yaml? This is so finicky..

Agreed, it's too finicky. Although instead of yaml, which is also finicky. we could go for toml. For example rust & go use that for their configs, and it's much nicer. Close to ini file syntax, but with better options.
